qa/L0_response_cache/test.sh

```diff
@@ -140,52 +171,77 @@ function check_server_expected_failure {
     fi
 }
 
-MODEL_DIR="${PWD}/models"
-mkdir -p "${MODEL_DIR}/decoupled_cache/1"
-mkdir -p "${MODEL_DIR}/identity_cache/1"
-
-# Check that server fails to start for a "decoupled" model with cache enabled
-EXTRA_ARGS="--model-control-mode=explicit --load-model=decoupled_cache"
-
-SERVER=/opt/tritonserver/bin/tritonserver
-SERVER_ARGS="--model-repository=${MODEL_DIR} --response-cache-byte-size=8192 ${EXTRA_ARGS}"
-SERVER_LOG="./inference_server.log"
-source ../common/util.sh
-run_server
-if [ "$SERVER_PID" != "0" ]; then
-    echo -e "\n***\n*** Failed: $SERVER started successfully when it was expected to fail\n***"
-    cat $SERVER_LOG
+function check_server_failure_decoupled_model {
+  MODEL_REPOSITORY="${1}"
+  MODEL="${2}"
+  EXTRA_ARGS="--model-control-mode="explicit" --load-model=${MODEL}"
+  SERVER_ARGS="--model-repository=${MODEL_REPOSITORY} --cache-config local,size=10480 ${EXTRA_ARGS}"
+  source ../common/util.sh
+
+  rm -f ${SERVER_LOG}
+  run_server
+  if [ "${SERVER_PID}" != "0" ]; then
+    echo -e "\n***\n*** Failed: ${SERVER} started successfully when it was expected to fail\n***"
+    cat ${SERVER_LOG}
     RET=1
 
-    kill $SERVER_PID
-    wait $SERVER_PID
-else
+    kill ${SERVER_PID}
+    wait ${SERVER_PID}
```

Comment:
Why do we need curly brackets?

Reply:
It's not required. Followed the style from lines 158,159 which was already present in the file.

Reply:
Could you please remove curly brackets here and everywhere where they are not needed?

Reply:
I'm pro curly brackets 😁 maybe we should leave it as is and add a bash linter to avoid ambiguity in style preference?

Reply:
We need a style to stick to

Reply:
My preference stems from past use of [shellcheck](https://github.com/koalaman/shellcheck): https://www.shellcheck.net/wiki/SC2250 - and I extend it to just preferring the brackets everywhere for consistency rather than only in problematic places. But it would be a separate effort to standardize all files to one or the other, they're currently mixed throughout I believe

Reply:
deferring this to later if that's OK with you @oandreeva-nv , looking to get this PR in

Reply:
I'm ok